website/static/js/koHelpers.js

```diff
@@ -44,6 +44,15 @@ var makeRegexValidator = function(regex, message, match) {
     };
 };
 
+var cleanURL = function(value) {
```

Comment:
I don't think so.  I am trimming the value before run it through the "http://"  This gets rid of an error where if you put a few spaces in before a website it will construct 'http://     google.com"

There might be a better implementation strategy though.  I am definitely open to suggestions.